### ISSUE TITLE
UBnormal - hr_bool_masks

### ISSUE THREAD

**Issue by jakubmicorek** [aleflabo/MoCoDAD#3]
Hi, thank you for your work!
At your provided GDRive link you provide the Avenue, HR-ShanghaiTech and UBnormal data.
Are the test_frame_masks already HR-related or not? 

Also according to you code in `get_hr_ubnormal_mask()` in the line `path_to_boolean_masks = f'./data/UBnormal/hr_bool_masks/{split}/test_frame_mask/*'` the folder `hr_bool_masks` should be in the UBnormal dataset. However, there is no such folder in the GDRive download.

Thank you in advance!

**Comment by aleflabo** [comment]
Hi Jakub,

We've checked the GDrive folder and the folder wasn't there. We've added it and now everything should work. Thanks for letting us know!

Regarding the first question, the test_frame_masks for Avenue and UBnormal are not the HR versions (the filter is applied in the postprocessing function https://github.com/aleflabo/MoCoDAD/blob/d10bcac5fb50779da1c3de61e274946370dbdba4/models/mocodad.py#L403C1-L404C1). HR-STC is already the HR version of the dataset.

If you have any doubts feel free to share them with us! 

Alessandro
